install.sh

```diff
@@ -897,7 +898,7 @@ case $platform in
     filetype="rpm"
     platform_package="el"
     # For Amazon Linux 2023 and onwards we can use the 'amazon' packages created instead of 'el' packages
-    if (( $platform_version >= 2023 )); then
+    if (( $platform_version == 2023 || $platform_version == 2 )); then
```

Comment:
I think it'd be good to keep this future proofed so that we don't have to update this again when amazon 2025 or whatever comes out

```suggestion
    if (( $platform_version >= 2023 || $platform_version == 2 )); then
```